internal/pkg/cli/storage_init.go

```diff
@@ -580,7 +574,7 @@ environment variables for easy access.`,
 	}
 	cmd.Flags().StringVarP(&vars.storageName, nameFlag, nameFlagShort, "", storageFlagDescription)
 	cmd.Flags().StringVarP(&vars.storageType, storageTypeFlag, typeFlagShort, "", storageTypeFlagDescription)
-	cmd.Flags().StringVarP(&vars.storageSvc, svcFlag, svcFlagShort, "", storageServiceFlagDescription)
+	cmd.Flags().StringVar(&vars.workloadName, forFlag, "", storageForFlagDescription)
```

Comment:
Should we have a shorthand for this `--for` flag since it is a required flag?

Reply:
Changed to workload.